store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -403,25 +431,26 @@ protected void addAdditionalOperationHandlers() {
                         && !AddElements.class.equals(op)
                         && !AddNamedOperation.class.equals(op)
                         && !AddNamedView.class.equals(op))
-                .forEach(op -> addOperationHandler(op, new FederatedOperationHandler()));
+                .forEach(op -> addOperationHandler(op, new FederatedNoOutputHandler()));
 
-        addOperationHandler(GetSchema.class, new FederatedGetSchemaHandler());
+        addOperationHandler(GetSchema.class, new FederatedGetSchemaHandler()); //TODO FS Likely to be deleted after Default Merge Mapping
 
-        addOperationHandler(Filter.class, new FederatedFilterHandler());
-        addOperationHandler(Aggregate.class, new FederatedAggregateHandler());
-        addOperationHandler(Transform.class, new FederatedTransformHandler());
+        addOperationHandler(Filter.class, new FederatedFilterHandler()); //TODO FS Likely to be deleted after Default Merge Mapping
```

Comment:
How is this different to a normal filter? Does each subGraph do the filtering locally?

Reply:
I do know fro memory I'd need to check, with the handler. but it would likely be removed because the default merging could take control and do what the handler is probably doing to combine the results.

Reply:
It literally wraps the operation for federation and getSchema. which is why it will likely will get replaced. or thought about.